man/systemd.time.xml

```diff
@@ -57,14 +57,13 @@
   <refsect1>
     <title>Displaying Time Spans</title>
 
-    <para>Time spans refer to time durations. On display, systemd will
-    present time spans as a space-separated series of time values each
-    suffixed by a time unit.</para>
+    <para>Time spans refer to time durations. On display, systemd will present time spans as a space-separated series
+    of time values each suffixed by a time unit. Example:</para>
 
     <programlisting>2h 30min</programlisting>
 
-    <para>All specified time values are meant to be added up. The
-    above hence refers to 150 minutes.</para>
+    <para>All specified time values are meant to be added up. The above hence refers to 150 minutes. Display is
+    locale-independent, only English language units are used.</para>
```

Comment:
"units" is strange in this context, "Tue" is not a unit, but a name. Also "Egnlish units" sounds like it was referring to imperial units or other such craziness. Maybe "only English names for the time units are accepted".